### DISCUSSION TITLE
Triggers for Custom profiles

### DISCUSSION THREAD

**Discussion by Azyyyyyy** [FluxpointDev/MultiRPC#19] [Old]
This was a planned feature for the original MultiRPC but due to it making the application unstable (from how the application as a whole was created) this is now focused for the rewrite

# How would this work
This would be an extension to the existing Custom page (As shown in the current version of the application, just in a disabled state)

There would be three kind of triggers
* Program opening/becoming the focused application
* It becomes x time for you (Example would be showing a dinner presence showing when it's 6pm)
    * On top of this they could be a time frame for this, going back to whatever presence they was when the time frame is over
* Timer for a presence which would be started (and could be paused, stopped) by the user, going back to whatever presence they was when the timer finishes

**Comment by xXBuilderBXx** [comment]
Redoing MultiRPC
